plugins/modules/consul_agent_check.py

```diff
@@ -74,33 +74,33 @@
     elements: str
   ttl:
     description:
-      - Checks can be registered with a TTL instead of a O(args) and O(interval) this means that the service will check in
-        with the agent before the TTL expires. If it does not the check will be considered failed. Required if registering
-        a check and the script an interval are missing Similar to the interval this is a number with a V(s) or V(m) suffix
-        to signify the units of seconds or minutes, for example V(15s) or V(1m). If no suffix is supplied V(s) will be used
-        by default, for example V(10) will be V(10s).
+      - Checks can be registered with a TTL instead of a O(args) and O(interval) this means that the service checks in with
+        the agent before the TTL expires. If it does not the check is considered failed. Required if registering a check and
+        the script an interval are missing Similar to the interval this is a number with a V(s) or V(m) suffix to signify
+        the units of seconds or minutes, for example V(15s) or V(1m). If no suffix is supplied V(s) is used by default, for
+        example V(10) is equivalent to V(10s).
       - Mutually exclusive with O(args), O(tcp) and O(http).
     type: str
   tcp:
     description:
-      - Checks can be registered with a TCP port. This means that Consul will check if the connection attempt to that port
+      - Checks can be registered with a TCP port. This means that Consul __FIXME__(will) check if the connection attempt to that port
```

Comment:
Here's a FIXME left.

Reply:
I think here "will" is actually good, since the check that Consul does will happen somewhen in the future, and not while the module is executed.
```suggestion
      - Checks can be registered with a TCP port. This means that Consul will check if the connection attempt to that port
```